ops/testing.py

```diff
@@ -11,8 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Framework for unit testing charms in a simulated environment.
 
-"""Infrastructure to build unit tests for charms using the ops library."""
+The module provides:
+
+- :class:`ops.testing.Harness`, a class to set up the environment for charms, and its:
+
+  - :meth:`~ops.testing.Harness.begin_with_initial_hooks` convenience method for declarative
+    test setup.
+  - individual :meth:`~ops.testing.Harness.begin` and :meth:`~ops.testing.Harness.cleanup`
+    methods to start and end the testing lifecycle.
+  - :meth:`~ops.testing.Harness.evaluate_status` method, which aggregates the
+    status of the charm after test interactions.
+  - :attr:`~ops.testing.Harness.model` attribute, which exposes e.g. the
+    :attr:`~ops.Model.unit` attribute for detailed assertions on the unit's state.
+
+.. note::
+    Unit testing is only one aspect of a comprehensive testing strategy. For more
+    on testing charms, see `Testing Strategies <https://juju.is/docs/sdk/testing>`_.
```

Comment:
see [Charm SDK | Testing](https://juju.is/docs/sdk/testing).

Reply:
Ugh the reference page title is actually `Juju | Testing` but I guess your take is better.